e2e/tests-dfx/identity_command.bash

```diff
@@ -132,6 +132,19 @@ teardown() {
     assert_command_fail dfx identity remove charlie
 }
 
+@test "identity remove: only remove identities with configured wallet if --drop-wallets is specified" {
+    WALLET="rwlgt-iiaaa-aaaaa-aaaaa-cai"
```

Comment:
Rather than hardcoding the wallet id, we can read it with `dfx identity get-wallet`